bsonnumpy/bson/bson-config.h

```diff
@@ -116,10 +142,25 @@
 #endif
 
 
+/*
+ * Define to 1 if you have _set_output_format (VS2013 and older).
+ */
+#if defined(_MSV_VER) && (_MSC_VER < 1900)
+# define BSON_NEEDS_SET_OUTPUT_FORMAT 1
+#endif
+
+#if BSON_NEEDS_SET_OUTPUT_FORMAT != 1
+# undef BSON_NEEDS_SET_OUTPUT_FORMAT
+#endif
```

Comment:
We can remove BSON_NEEDS_SET_OUTPUT_FORMAT. It's no longer present in libbson 1.17.0

Reply:
Done.